### PR TITLE
Migrate from flask_githubapp to gidgethub

### LINKED ISSUE #23: [1pt] migrate over to using gidgethub as it is better maintained
Switch to start using [gidgethub](https://gidgethub.readthedocs.io/en/latest/sansio.html#module-gidgethub.sansio) instead of the current github interfacing module. This is because the gidgethub is more actively maintained.

**Acceptance Criteria**
- [ ] switch Python modules
- [ ] confirm bot is working the same

### PR DESCRIPTION
**Pull request by shreekarSS**
Signed-off-by: Shreekara <sshreeka@redhat.com>

## Related Issues and Dependencies

Fixes: https://github.com/thoth-station/ttm-as-a-service/issues/23

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -1,50 +1,79 @@
...
-@github_app.on("installation.deleted")
-def uninstall_app():
-    print("removed")
+if __name__ == "__main__":
+    app = web.Application()
+    app.router.add_routes(routes)
+    port = os.environ.get("PORT")
+    if port is not None:
+        port = int(port)
+    web.run_app(app, port=port)
```
Comment (Gkrumbach07) [inline app.py:80]:
```suggestion
web.run_app(app, port=port)
```

unindent last line

File: app.py [hunk 2]
```diff
@@ -1,50 +1,79 @@
...
+        body = await request.read()
+        event = sansio.Event.from_http(request.headers, body, secret=GITHUBAPP_SECRET)
+        async with aiohttp.ClientSession() as session:
+            gh = gh_aiohttp.GitHubAPI(session, "demo", cache=cache)
+            await asyncio.sleep(1)
+            await router.dispatch(event, gh)
+        return web.Response(status=200)
+    except Exception as ex:
+        print(ex)
+        return web.Response(status=500)
```
Comment (Gkrumbach07) [inline app.py]: Is this route needed?

**Comment by Gkrumbach07** [comment]
Other changes for migrating over to using `aiohttp`:

- [ ] You will have to remove `wsgi.py` and `gunicorn.conf.py` as it is no longer used.
- [ ] change [port](https://github.com/thoth-station/ttm-as-a-service/blob/main/manifests/base/deploymentconfig.yaml#L26) to 8080 now
    - [ ] also [here](https://github.com/thoth-station/ttm-as-a-service/blob/main/manifests/base/deploymentconfig.yaml#L34)
    - [ ] and [here](https://github.com/thoth-station/ttm-as-a-service/blob/main/manifests/base/service.yaml#L9-L11)
- [ ] dont need [this](https://github.com/thoth-station/ttm-as-a-service/blob/main/manifests/base/deploymentconfig.yaml#L31-L32)
- [ ] update [Pipfile](https://github.com/thoth-station/ttm-as-a-service/blob/main/Pipfile#L7) and Pipfile lock. need to add/remove new old modules used.

File: app.py (hunk 1 above)
Comment (shreekarSS) [reply app.py:80]: ack

**Review by Gkrumbach07 — APPROVED** [review]

**Pull request merged**